infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -85,6 +87,16 @@ export class AtatWebApiStack extends cdk.Stack {
     // Prevent the GET function from being able to write to DynamoDB (it doesn't need to)
     table.grantReadData(getPortfolioDraftsFn);
 
+    // createPortfolioStep
```

Comment:
```suggestion
    // operationId: createPortfolioStep
```

Reply:
This was originally just direct output from grepping the API yaml.